perfkitbenchmarker/packages/fio.py

```diff
@@ -82,22 +66,51 @@ def ParseJobFile(job_file):
     A dictionary of dictionaries of sample metadata, using test name as keys,
         dictionaries of sample metadata as value.
   """
-  parameter_metadata = {}
+  job_file = job_file.replace(JOB_STONEWALL_PARAMETER, '')
+  config = ConfigParser.ConfigParser()
+  config.readfp(io.BytesIO(job_file))
+  sections = config.__dict__['_sections']
```

Comment:
Prefer the public interface: `.sections()` gives a list of section names; `.items(section_name)` gives a list of key-value pairs in a section.